test/main/provider/index.test.js

```diff
@@ -20,6 +20,11 @@ let accountRequests = []
 jest.mock('../../../main/store')
 jest.mock('../../../main/chains', () => ({ send: jest.fn(), syncDataEmit: jest.fn(), on: jest.fn() }))
 jest.mock('../../../main/accounts', () => ({}))
+jest.mock('../../../main/reveal', () => ({
+  resolveEntityType: jest.fn(async () => {
```

Comment:
can simplify this to
```suggestion
  resolveEntityType: jest.fn().mockResolvedValue('NATIVE_TRANSFER')
```

Reply:
also probably best to refer to the type itself rather than using the string literal

Reply:
actually is this even the correct return value for this function? in the code looks like it can only return 'unknown', 'contract' or 'external'